install/linux/usr/share/odemis/sim/secom2-sem-ctrl-sim.odm.yaml

```diff
@@ -0,0 +1,221 @@
+# sudo modprobe comedi_test
+# sudo comedi_config /dev/comedi0 comedi_test 1000000,1000000
+"SEM Scan Interface": {
+    class: semcomedi.SEMComedi,
+    role: null,
+    init: {device: "/dev/comedi0"},
+    # more detectors can be added, if necessary
+    children: {
+       scanner: "SEM E-beam",
+       detector0: "SEM Detector",
+    }
+}
+
+# Connect:
+# X -> AO 0
+# Y -> AO 1
+# Ground -> AO GND
+"SEM E-beam": {
+    # Internal child of SEM ExtXY, so no class
+    creator: "SEM Scan Interface",
+    role: null,
+    init: {
+        channels: [0, 1],
+        max_res: [4096, 3072], #px, to force a 4:3 ratio
+        limits: [[-3.70, 3.68], [-3.6, 3]],  # V
+        park: [-4.3, -4.3], # V
+        # Digital output port mapping on the Delmic scanning box v2:
+        # 0 = Relay
+        # 1 = Open drain output (Y0.0)
+        # 2 = Digital Out 1
+        # 3 = Digital Out 0
+        # 4 = Status led
+        scanning_ttl: {0: [True, "external"], 4: True}, # output ports -> True (indicate scanning) or False (indicate parked)
+        settle_time: 10.e-6, # s
+        hfw_nomag: 0.3457,
+    },
+}
+
+# Must be connected on AI1/AI9 (differential)
+"SEM Detector": { # aka ETD
+    # Internal child of SEM Scan Interface, so no class
+    role: se-detector,
+    init: {
+        channel: 0,
+        limits: [-2, 2], # V
+    },
+}
+
+"SEM via API": {
+    class: zeiss.SEM,
+    role: null,
+    init: {
+        port: "/dev/fake",
+        #port: "/dev/ttyUSB*",
+    },
+    children: {
+        scanner: "EBeam control",
+        focus: "EBeam Focus",
+        stage: "Sample Stage",
+    }
+}
+
+"EBeam control": {
+    creator: "SEM via API",
+    role: null,
+    init: {
+        hfw_nomag: 0.114,  # m, theoretical HFW if mag == 1x
+    },
+}
+
+"EBeam Focus": {
+    # Internal child of zeiss.SEM, so no class
+    role: ebeam-focus,
+}
+
+"Optical Objective": {
+    class: static.OpticalLens,
+    role: lens,
+    init: {
+       mag: 40.0, # ratio, (actually of the complete light path)
+       na: 0.95, # ratio, numerical aperture
+       ri: 1.0, # ratio, refractive index
+    },
+    affects: ["Camera"]
+}
+
+# Semrock FF01-440/521/607/700
+"Optical Emission Filter": {
+    class: static.LightFilter,
+    role: filter,
+    init: {band: [[420.e-9, 460.e-9],
+                  [510.e-9, 532.e-9],
+                  [590.e-9, 624.e-9],
+                  [677.e-9, 723.e-9]]}, # m,m
+    affects: ["Camera"]
+}
+
+# Axes: X is horizontal on screen (going left->right), physical: far->close when looking at the door
+#       Y is vertical on screen (going bottom->top), physical: left->right when looking at the door
+"Camera": {
+    class: andorcam2.FakeAndorCam2,
+    role: ccd,
+    init: {
+        device: 0,
+        image: "andorcam2-fake-clara.tiff",
+    },
+    properties: {
+        exposureTime: 0.3, # s
+        # targetTemperature: -40, # °C
+#        resolution: [2048, 2048],  # To crop Y axis
```

Comment:
Those two lines should stay? Can you add a comment if yes why they are commented?